bitbots_basler_camera/config/camera_calibration_jack.yaml

```diff
@@ -1,17 +1,17 @@
 image_width: 2048
 image_height: 1536
-camera_name: jack
+camera_name: narrow_stereo
 camera_matrix:
   rows: 3
   cols: 3
-  data: [1338.64532,    0.     , 1026.12387,
-            0.     , 1337.89746,  748.42213,
+  data: [1924.06561,    0.     , 1030.80485,
```

Comment:
We might want to look at this values as they are quite different compared to the other ones.

Reply:
So the calibration of jack seems to be a bit off.